vendor/blockchain_contracts/src/bin/calculate_offsets/ethereum/rfc003/metadata.rs

```diff
@@ -0,0 +1,15 @@
+pub struct Metadata {
+    pub ledger_name: String,
+    pub asset_name: String,
+    pub contract_hex: String,
+}
+
+impl Metadata {
+    pub fn new(ledger_name: String, asset_name: String, contract_hex: String) -> Metadata {
```

Comment:
This seems unnecessary if all the fields are public anyway.

Reply:
removed `new`